pkg/alb/loadbalancer/loadbalancer.go

```diff
@@ -597,10 +596,12 @@ func createLBName(namespace string, ingressName string, clustername string) stri
 	hasher := md5.New()
 	hasher.Write([]byte(namespace + ingressName))
 	hash := hex.EncodeToString(hasher.Sum(nil))[:4]
+
+	r, _ := regexp.Compile("[[:^alnum:]]")
```

Comment:
I think we will need to catch the error here

Reply:
Apologies for the late reply. I must have missed the notification.

Just checking - what would you like me to do with the error? Since we are compiling a static regex that we know compiles, there didn't seem to be much point in checking for an error, but I'm admittedly not very familiar with Go. Happy to adjust to whatever you like here.